src/main/java/Task.java

```diff
@@ -0,0 +1,17 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public void Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    public void markAsDone(){
```

Comment:
Avoid unnecessary use of this with fields.
Refer to: https://se-education.org/guides/conventions/java/index.html#variables